src/components/donation/Steps/CurrentStep.tsx

```diff
@@ -22,7 +25,7 @@ export default function CurrentStep({ config }: Props) {
     return (
       <Splits
         {...state}
-        liquidSplitPct={state.liquidSplitPct ?? config?.liquidSplitPct}
+        liquidSplitPct={liquidSplitPct}
```

Comment:
I tried to update preview slider (`88`)  - but when I click `update snippet`. The previev slider didn't follow (back to `50`)

![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/320a5b7a-6455-4f67-a44c-60ab00fbd219)

Reply:
This is a very weird bug. I updated [the useEffect responsible for updating the liquid %](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2819/files#diff-9c40ba9a8234eefaaca353ef2f5fb9fce9c968c7ba5459a1d78ac75780ac3235R41) and verified that the nested `setState` gets called:
```ts
  useEffect(() => {
    if (config?.liquidSplitPct) {
      setLiquidSplitPct(config.liquidSplitPct); // <-- this gets called!
    }
  }, [config, config?.liquidSplitPct]);
```
Despite this, the actual state value (`liquidSplitPct`) **does not get updated** :exploding_head: